internal/envoy/auth/filters.go

```diff
@@ -0,0 +1,42 @@
+// MIT License
+//
+// Copyright (c) 2022 Kubeshop
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package auth
+
+import (
+	// xds_core_v3 "github.com/cncf/xds/go/xds/core/v3"
+	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+const (
+	ResourceApiVersion  = envoy_config_core_v3.ApiVersion_V3
+	TransportApiVersion = envoy_config_core_v3.ApiVersion_V3
+)
+
+func TimeoutDefault() *durationpb.Duration {
+	return &durationpb.Duration{
+		Seconds: 16,
+	}
+	// // Envoy will wait indefinitely for the first xDS config.
+	// return durationpb.New(time.Second * 0)
```

Comment:
commented out code

Reply:
Can't we have this documented elsewhere?

Reply:
Where?

Shall I just remove the comment and the code block?

Reply:
Let's close this. It is taking more time than we want to